elements/pfe-icon/README.md

```diff
@@ -88,8 +88,21 @@ To updating an existing icon set, you use the same `addIconSet` function.  The f
 
 There are several powerful variables available to hook into and override default styles.
```

Comment:
```suggestion
There are several powerful ways to hook into and override default styles.
```

This heading says "several powerful variables" but then the next thing below it says "the color attribute" which could be confusing.